var/spack/repos/builtin/packages/py-radical-pilot/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyRadicalPilot(PythonPackage):
+    """RADICAL-Pilot is a Pilot system specialized in executing applications
+    composed of many computational tasks on high performance computing (HPC)
+    platforms."""
+
+    homepage = 'https://radical-cybertools.github.io'
+    git      = 'https://github.com/radical-cybertools/radical.pilot.git'
+    pypi     = 'radical.pilot/radical.pilot-1.6.7.tar.gz'
+
+    maintainers = ['andre-merzky']
+
+    version('devel', branch='devel')
+    version('1.6.7', sha256='6ca0a3bd3cda65034fa756f37fa05681d5a43441c1605408a58364f89c627970')
+
+    depends_on('py-radical-utils',       type=('build', 'run'))
+    depends_on('py-radical-saga',        type=('build', 'run'))
+    depends_on('py-radical-utils@devel', type=('build', 'run'), when='@devel')
+    depends_on('py-radical-saga@devel',  type=('build', 'run'), when='@devel')
```

Comment:
I don't think develop is actually required is it?

Reply:
it is provided in case user wants to set a version with latest changes, thus all `devel` (`develop`) versions will be used instead of releases (developed versions are synced)

Reply:
But users can always get that with `spack install py-radical-pilot@develop ^py-radical-utils@develop ^py-radical-saga@develop`. `depends_on` lists hard requirements, not soft preferences.

Reply:
Thank you! yeah, missed that, will remove these dependencies